lib/src/cli/deployapps/deleteapps_command.dart

```diff
@@ -0,0 +1,84 @@
+////////////////////////////////////////////////////////////////////////////////
+//
+// Copyright 2022 Realm Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+////////////////////////////////////////////////////////////////////////////////
+
+import 'dart:async';
+import 'dart:io';
+
+import 'package:args/command_runner.dart';
+
+import 'options.dart';
+import 'baas_client.dart';
+
+class DeleteAppsCommand extends Command<void> {
+  @override
+  final String description = 'Delete test applications from MongoDB Atlas.';
+
+  @override
+  final String name = 'delete-apps';
+
+  @override
+  bool get hidden => true;
+  
+  late Options options;
+
+  DeleteAppsCommand() {
+    populateOptionsParser(argParser).addOption("appIds", help: "List of appIds of deployed apps on MongoDB Atlas.");
+  }
+
+  @override
+  FutureOr<void>? run() async {
+    options = parseOptionsResult(argResults!);
+
+    var result = argParser.parse(argResults!.arguments);
```

Comment:
That doesn't seem correct to me - why not create dedicated options class for this command?

Reply:
Yes, this folder and commands might be restructured.

Reply:
I changed the name of the folder from deployapps to atlas_apps then I used the same Options for both commands, since after refactoring to use the differentiator both commands have the same options.